conf/messages

```diff
@@ -2,10 +2,9 @@ invalid.action=Review the application
 override.scopes.required=Scopes are required when enabling this access override
 override.scopes.incorrect=You must choose at least one correct scope
 scopes.required=Scopes are required
-whitelistedIp.required=Required field
-whitelistedIp.invalid=Invalid value
-whitelistedIp.invalid.private=Private network range
-whitelistedIp.invalid.range=Out of range
+whitelistedIp.invalid= {0} is an invalid value
+whitelistedIp.invalid.private= {0} is a private network range
+whitelistedIp.invalid.range= {0} is out of range
```

Comment:
Other than the private network range one, not sure the other validation messages make sense in isolation

Reply:
Maybe we could add some extra info about the range (we're limiting the mask part of the CIDR block so it can't go below 24)